Regen HTTP connector documentation (#25)
aiven/aiven-kafka-connect-http: docs/sink-connector-config-options.rst

  * Default: null
  * Importance: low

``oauth2.access.token.url``

>   The URL to be used for fetching access token. Client Credentials is only supported grand type.

Reviewer: ```suggestion
  The URL to be used for fetching an access token. Client Credentials is the only supported grant type.
```

  * Type: string
  * Default: null
  * Valid Values: HTTP(S) ULRs
  * Importance: high
  * Dependents: ``oauth2.client.id``, ``oauth2.client.secret``, ``oauth2.client.authorization.mode``, ``oauth2.client.scope``, ``oauth2.response.token.property``

``oauth2.client.id``

>   The client id used for fetching access token.

Reviewer: ```suggestion
  The client id used for fetching an access token.
```

  * Type: string
  * Default: null
  * Valid Values: OAuth2 client id
  * Importance: high
  * Dependents: ``oauth2.access.token.url``, ``oauth2.client.secret``, ``oauth2.client.authorization.mode``, ``oauth2.client.scope``, ``oauth2.response.token.property``

``oauth2.client.secret``

>   The secret used for fetching access token.

Reviewer: ```suggestion
  The secret used for fetching an access token.
```

  * Type: password
  * Default: null
  * Importance: high
  * Dependents: ``oauth2.access.token.url``, ``oauth2.client.id``, ``oauth2.client.authorization.mode``, ``oauth2.client.scope``, ``oauth2.response.token.property``

``oauth2.client.authorization.mode``

>   Specifies how to encode client_id and client_secret in the OAuth2 authorization request. If set to 'header', the credentials are encoded as an 'Authorization: Basic <base-64 encoded client_id:client_secret>' HTTP header. If set to ‘url’, then client_id and client_secret are sent as URL encoded parameters. Default is 'header'

Reviewer: ```suggestion
  Specifies how to encode ``client_id`` and ``client_secret`` in the OAuth2 authorization request. If set to ``header``, the credentials are encoded as an ``Authorization: Basic <base-64 encoded client_id:client_secret>`` HTTP header. If set to ``url``, then ``client_id`` and ``client_secret`` are sent as URL encoded parameters. Default is ``header``.
```

  * Type: string
  * Default: HEADER
  * Valid Values: HEADER,URL
  * Importance: medium
  * Dependents: ``oauth2.access.token.url``, ``oauth2.client.id``, ``oauth2.client.secret``, ``oauth2.client.scope``, ``oauth2.response.token.property``

``oauth2.client.scope``

>   The scope used for fetching access token.

Reviewer: ```suggestion
  The scope used for fetching an access token.
```

  * Type: string
  * Default: null
  * Valid Values: OAuth2 client scope
  * Importance: low
  * Dependents: ``oauth2.access.token.url``, ``oauth2.client.id``, ``oauth2.client.secret``, ``oauth2.client.authorization.mode``, ``oauth2.response.token.property``

``oauth2.response.token.property``

>   The name of the JSON property containing the access token returned by the OAuth2 provider. Default value is 'access_token'.

Reviewer: ```suggestion
  The name of the JSON property containing the access token returned by the OAuth2 provider. Default value is ``access_token``.
```

Delivery
^^^^^^^^

``kafka.retry.backoff.ms``

>   The retry backoff in milliseconds. This config is used to notify Kafka Connect to retry delivering a message batch or performing recovery in case of transient exceptions. Maximum value is 86400000 (24 hours).

Reviewer: ```suggestion
  The retry backoff in milliseconds. This config is used to notify Kafka Connect to retry delivering a message batch or performing recovery in case of transient failures.
```
